Source/Core/HeadingPitchRoll.js

```diff
@@ -40,17 +42,23 @@ define([
             throw new DeveloperError('quaternion is required');
         }
         //>>includeEnd('debug');
+
+        deprecationWarning('HeadingPitchRoll.fromQuaternion', 'This function now uses a counter-clockwise orientation as per mathematical conventions. This deprecation warning will be removed in Cesium 1.40.');
```

Comment:
Submit an issue to remove this warning.

Also, make the warning more precise.  This started in 1.38 and users just need to add the negative sign.

Reply:
Same comment throughout.